api2/pkg/webapp/templates/casenew.gohtml

```diff
@@ -29,6 +29,22 @@
                         </select>
                         <label for="case-type">Select Case Type</label>
                     </div>
+
+                    <div class="form-floating mb-3">
+                        <select disabled id="team" class="form-select" name="teamIdSelect">
+                            {{if .TeamList}}
+                                <option {{if not $.TeamID}}selected{{end}} value="" hidden="hidden">Choose</option>
+                                {{range .TeamList.Items}}
+                                    <option {{if eq .ID $.TeamID}}selected{{end}} value="{{.ID}}">{{.Name}}</option>
+                                {{end}}
+                            {{else}}
+                                <option selected disabled>No parties defined</option>
+                            {{end}}
+                        </select>
+                        <label for="team">Select Team</label>
+                    </div>
+                    <input id="teamId" name="teamId" type="hidden" value="{{.TeamID}}"/>
+
```

Comment:
In this section, the Team has to be bound to the CaseType. 
Since CaseTypes are owned by teams

2 options:
1. Make the Team readonly. When the user changes the CaseType, then set the team using javascript
2. When the user changes the team, filter the CaseType list to only show the case types for that team

Reply:
Hey @ludydoo those requirements are met by the current implementation.

We asked Nico for advice, since this page redirects the user each time the case type is changed (it adds the caseTypeId as a URL param and the backend sends it back down when the template is re-rendered).

We followed a similar approach with the TeamID, we used JS to map the caseTypeId to a teamId and then we attach that to the URL sent with the redirect.

The following shows how it works:
https://user-images.githubusercontent.com/15915453/123253597-ed63a400-d4ed-11eb-8ad1-6f2712cd0871.mp4

Reply:
I see, then I think there is no need for the teamId request parameter, as the teamId is already inferred from the caseType ?

Reply:
```javascript
const caseTypeTeamMap = {
            {{range .CaseTypes.Items}}
            "{{.ID}}": "{{.TeamID}}",
            {{end}}
        }

        const caseTypeSelect = document.querySelector("#case-type");
        caseTypeSelect.onchange = (event) => {
            const caseTypeId = event.target.value;
            if (caseTypeId === "") {
                return;
            }
            window.location.href = `/cases/new?caseTypeId=${caseTypeId}&teamId=${caseTypeTeamMap[caseTypeId]}`
        }
```

Reply:
You are right @ludydoo, we could actually just get the team info server side and send it down!